build_runner/test/common/in_memory_reader.dart

```diff
@@ -7,22 +7,16 @@ import 'dart:mirrors';
 import 'package:build/build.dart';
 import 'package:build_runner/build_runner.dart';
 import 'package:build_test/build_test.dart';
+import 'package:glob/glob.dart';
 
 class InMemoryRunnerAssetReader extends InMemoryAssetReader
     implements RunnerAssetReader {
   InMemoryRunnerAssetReader([Map<AssetId, DatedValue> sourceAssets])
       : super(sourceAssets);
 
   @override
-  Stream<AssetId> listAssetIds(Iterable<InputSet> inputSets) async* {
-    for (var id in assets.keys) {
-      var matches = inputSets.any((inputSet) {
-        if (inputSet.package != id.package) return false;
-        return inputSet.globs.any((glob) => glob.matches(id.path));
-      });
-      if (matches) yield id;
-    }
-  }
+  Iterable<AssetId> findAssets(Glob glob, {String packageName}) => assets.keys
```

Comment:
I believe all implementations of AssetReader today *could* have a default package.

With the upcoming change in `build` the `InMemoryAssetReader` is going to end up needing a default package. I think that should work out...